packages/core/content-releases/admin/src/pages/tests/ReleaseDetailsPage.test.tsx

```diff
@@ -9,6 +9,10 @@ jest.mock('@strapi/helper-plugin', () => ({
   ...jest.requireActual('@strapi/helper-plugin'),
   // eslint-disable-next-line
   CheckPermissions: ({ children }: { children: JSX.Element }) => <div>{children}</div>,
+  useRBAC: jest.fn().mockImplementation(() => ({
+    isLoading: false,
+    allowedActions: { canUpdate: true, canDelete: true },
+  })),
```

Comment:
can you add a test to check that we get the expect UI when the hook says we don't have the right permissions?